### DISCUSSION TITLE
Dependency Graph showing dependencies but not dependents

### DISCUSSION THREAD

**Discussion by Xwilarg** [community/community#23565] [New to GitHub]
<p>Hi,</p>
<p>So in one of my project the dependencies are working properly</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d8d4dc201f2c256281bdb7b5544ca71c80a2b3f2.png" data-download-href="/uploads/short-url/uWbboqFPzPANmnZ09pcl3Ta7GXU.png?dl=1" title="dependencies.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/20053454/181077927-4fd630ac-7842-45ca-99cd-b9b1eacec9cd.png" alt="dependencies.png" data-base62-sha1="uWbboqFPzPANmnZ09pcl3Ta7GXU" width="690" height="231" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d8d4dc201f2c256281bdb7b5544ca71c80a2b3f2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">dependencies.png</span><span class="informations">740×248 8.78 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>But the dependents tab is empty</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/24eaf3f828c2ce62abeffddb968dbcb8151dbd73.png" data-download-href="/uploads/short-url/5gAzyg5809cnf7kugJT20BrG9iP.png?dl=1" title="dependents.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/20053454/181077928-a7d1381c-7f4c-401f-aaf4-51994237b26a.png" alt="dependents.png" data-base62-sha1="5gAzyg5809cnf7kugJT20BrG9iP" width="690" height="323" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/24eaf3f828c2ce62abeffddb968dbcb8151dbd73_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">dependents.png</span><span class="informations">719×337 8.46 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>(The project being on NuGet I know that some people already downloaded it and I myself have another project that use it.)</p>
<p>Further more, the dependents tab is working for another one of my project, both of them are made with the same language (C#)</p>
<p>I was wondering if one of you had any idea of why this was happening and how I could solve it,</p>
<p>Thanks in advance,</p>

**Comment by AndreaGriffiths11** [comment]
<p>HI <a class="mention" href="/xwilarg">@xwilarg</a>,</p>
<p>Thanks for being here. Which repository should the package be linked to? Drop the link and I’ll take a look, please.</p>

**Comment by Xwilarg** [comment]
<p>Thanks for your answer,</p>
<p>The repository is I’m having issues with is <a href="https://github.com/Xwilarg/BooruSharp/network/dependents" rel="nofollow noopener">https://github.com/Xwilarg/BooruSharp/network/dependents</a></p>
<p>And here’s a repository who use it: <a href="https://github.com/Xwilarg/Sanara/network/dependencies" rel="nofollow noopener">https://github.com/Xwilarg/Sanara/network/dependencies</a> </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks for the follow up. I’ve run a job to redetect the manifests of that repository, and it looks like the network dependents list has been updated. Do things look as expected for you now?</p>

**Comment by Xwilarg** [comment]
<p>Thanks a lot for your help, looks like everything is working perfectly now!</p>

**Comment by wang0618** [comment]
<p>Same problem on repository <a href="https://github.com/wang0618/PyWebIO" rel="nofollow noopener">https://github.com/wang0618/PyWebIO</a> . Would you fix it for me. Thanks</p>

**Comment by arthurfiorette** [comment]
<p>Same with <a href="https://github.com/ArthurFiorette/sink-library/network/dependents" class="inline-onebox" rel="noopener nofollow ugc">Network Dependents · ArthurFiorette/sink-library · GitHub</a><br>
and <a href="https://github.com/ArthurFiorette/mysouls/network/dependencies" class="inline-onebox" rel="noopener nofollow ugc">Dependencies · ArthurFiorette/mysouls · GitHub</a></p>

**Comment by tomschr** [comment]
<p><a class="mention" href="/andreagriffiths11">@AndreaGriffiths11</a> Sorry to bother, but I saw this thread after I’ve searched it. It seems, this is exactly what I need. :wink:<br>
Unfortunately, I have the same issue with my repo, see here: <a href="https://github.com/python-semver/python-semver/network/dependents" class="inline-onebox" rel="noopener nofollow ugc">Network Dependents · python-semver/python-semver · GitHub</a></p>
<p>Could you have a look? Many thanks! :heart:</p>

**Reply by AndreaGriffiths11** [reply]
apologies for the delay Tom, take a look now and LMK

**Reply by tomschr** [reply]
No problem. It's solved now. Thanks!

**Comment by donkirkby** [comment]
@AndreaGriffiths11, could you please redetect the manifests of [svg-turtle]? It's referenced by [judge-turtle].
It looks like [zero-play] has a similar problem, referenced by [shibumi-games].

I don't know if it's relevant, but the dependencies of shibumi-games have a link to zero-play, whereas judge-turtle has svg-turtle with no link.

Thanks for helping everyone out with this. Is there any way for us to avoid this problem when we set up repositories?

[svg-turtle]: https://github.com/donkirkby/svg-turtle/network/dependents
[judge-turtle]: https://github.com/dodona-edu/judge-turtle/network/dependencies
[zero-play]: https://github.com/donkirkby/zero-play/network/dependents
[shibumi-games]: https://github.com/donkirkby/shibumi-games/network/dependencies

**Reply by donkirkby** [reply]
Since this discussion seemed to have gone cold, I sent a [GitHub support request], and they fixed both repositories within a day.

[GitHub support request]: https://support.github.com/contact/account

**Reply by AndreaGriffiths11** [reply]
My apologies for the longggg delay 🤦🏽‍♀️
So glad you had it resolved now!

**Comment by fabiocaccamo** [comment]
@AndreaGriffiths11 I'm experiencing the same issue with all these repos:

- https://github.com/fabiocaccamo/FCUUID

- https://github.com/fabiocaccamo/django-admin-interface

- https://github.com/fabiocaccamo/python-benedict

- https://github.com/fabiocaccamo/FCFileManager

- https://github.com/fabiocaccamo/django-colorfield

- https://github.com/fabiocaccamo/django-treenode

- https://github.com/fabiocaccamo/django-maintenance-mode

- https://github.com/fabiocaccamo/django-extra-settings

- https://github.com/fabiocaccamo/FCCurrentLocationGeocoder

- https://github.com/fabiocaccamo/python-fsutil

- https://github.com/fabiocaccamo/FCIPAddressGeocoder

- https://github.com/fabiocaccamo/django-freeze

- https://github.com/fabiocaccamo/python-codicefiscale

- https://github.com/fabiocaccamo/django-redirects

- https://github.com/fabiocaccamo/python-fontbro

- https://github.com/fabiocaccamo/utils.js


I think it's related, but also the "Used by" block in the repositories sidebar is no more visible.

**Comment by luwes** [comment]
Could you have a look at our repo as well? https://github.com/muxinc/elements/network/dependents Thank you!
